### ISSUE TITLE
New rule: input object type values have descriptions

### ISSUE THREAD

**Issue by cjoudrey** [cjoudrey/graphql-schema-linter#62]
As reported in https://github.com/cjoudrey/graphql-schema-linter/issues/55#issuecomment-346975694, there is currently no rule to ensure input object type values have a description.

Example:

```graphql
input User {
  username: String

  # Description
  withDescription: String
}
```

Rather than add this to the existing `fields-have-descriptions` rule, I think we might want to roll this out as a new rule such that users can opt in to it.

cc @mscharley

**Comment by cjoudrey** [comment]
I'll tackle this tomorrow.

**Comment by mscharley** [comment]
I don't necessarily disagree with it being it's own rule though my vote would be to have it in the default set of rules so that users would opt-out rather than opt-in. This way all fields must have descriptions by default and there's no surprises in the default configuration.

**Comment by cjoudrey** [comment]
:100: the default behaviour is to validate the schema against all rules.

If someone only wants a subset of the rules they can do so by specifying a whitelist via `--rules <rules>`. When someone uses this option, they will not _automatically_ get new rules. They will need to opt in to them. This might sound like a downside, but I've been told it makes upgrading the package easier as they don't need to fix their schema at the same time as upgrading.

A few months ago we have `--except <rules>`, but have since removed it to keep the command-line options simple.

We can definitely revisit all this in the future though. Nothing is set in stone.

**Comment by cjoudrey** [comment]
This rule has been added to `v0.0.20`.
